kod/util/assgame.kod

```diff
@@ -1125,11 +1130,21 @@ messages:
    IsViableGame()
    "If the game is viable (i.e. there are enough combatants) start the match."
    {
+      local i, poGift;
       if Length(plCombatants) > 1
       {
          return TRUE;
       }
 
+      % inform the lonely soul that the game is not viable.
+      for i in plCombatants
+      {
+         Send(i,@ReceiveNestedMail,#nest_list=[4,AssGame_not_viable],
+            #from=assgame_roq,#dest_list=[i]);
+         poGift = Create(&InkyCap,#number=2);
```

Comment:
I don't think we should add even this small gift.  Someone will find a way to game it.